frontend/openchat-client/src/events.ts

```diff
@@ -33,7 +33,7 @@ export class RemoteVideoCallStartedEvent extends CustomEvent<{
     messageId: bigint;
 }> {
     constructor(chatId: ChatIdentifier, userId: string, messageId: bigint) {
-        super("openchat_event", { detail: { chatId, userId, messageId } });
+        super("openchat_event", { detail: { chatId, userId, messageId: BigInt(messageId) } });
```

Comment:
We should just do this within `rtc.ts` where we receive the message.
We convert other BigInt fields in there too.